README.md

```diff
@@ -164,6 +164,28 @@ python util/taskcluster.py --target /tmp --source tensorflow --arch gpu --artifa
 pip install /tmp/tensorflow_gpu_warpctc-1.3.0rc0-cp27-cp27mu-linux_x86_64.whl
 ```
 
+### Common Voice training data
+
+The Common Voice corpus consists of voice samples that were donated by the [Common Voice web application](https://voice.mozilla.org/).
+For automatically downloading and importing the corpus into a given directory, you can call:
+
+```bash
+bin/import_cv.py path/to/a/directory
+```
+
+Please be aware that this requires at least 70GB of free disk space and quite some time to conclude. 
+As this process creates a huge number of small files, using a SSD drive is highly recommended.
+To some extend the importer is reentrant and tries its best to continue where it got interrupted. 
+Once the import is done, the directory will contain a bunch of CSV files:
+`cv-valid-train.csv`, `cv-valid-dev.csv` and `cv-valid-test.csv` are the official user-validated sets for training, validation and test respectively.
```

Comment:
Bullet point lists would be much more readable here:

"The following files are official user-validated sets for training, validation,....
* cv-valid-train.csv
* cv-valid-dev.csv
* blah.blah

The following files are non-validated un...
* csv1
* something-else.csv"

etc.